### PR TITLE
Update troubleshooting.asciidoc

### PR DESCRIPTION
**Pull request by leahleahy**

### CODE REVIEW

File: docs/troubleshooting.asciidoc [hunk 1]
```diff
@@ -2,8 +2,8 @@
 == Troubleshooting
 
 Is something not working as expected?
-Don't worry if you can't figure out what the problem is; we’re here to help!
-As a first step, ensure your app is compatible with the agent's <<supported-tech,supported technologies>>.
+Don't worry if you can't figure out what the problem is, we’re here to help!
```
Comment (EamonnTP) [inline docs/troubleshooting.asciidoc:5]: I would keep the semicolon.

**Review by EamonnTP — CHANGES_REQUESTED** [review]
Looks great. Just one minor suggestion.

File: docs/troubleshooting.asciidoc (hunk 1 above)
Comment (dedemorton) [reply docs/troubleshooting.asciidoc:5]: Incidentally here's a good topic about comma splices: https://www.grammarly.com/blog/comma-splice/

**Comment by bmorelli25** [comment]
Using the "edit" functionality to change branches is unfortunately too messy. I've cherry-picked the commits in this PR to a new branch and opened https://github.com/elastic/apm-agent-go/pull/951.

**Pull request closed without merging**